R/estimate_outcomes.R

```diff
@@ -12,8 +12,8 @@
 #' @return A `<data.frame>` with the columns in `data`, and with two additional
 #' columns:
 #'
-#'  - `"known_outcomes"` for the total number of known outcomes on that day
```

Comment:
here it could say "for the number of cases with a known outcome on x day", so it doesn't sound like it's cumulative

Reply:
Rephrasing as "for the number of cases with an outcome of interest (usually, death) estimated to be known on that day of the outbreak"